scripts/lisk/docker/mainnet/containerStart_rebalancer.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+. $(dirname $(realpath "$0"))/sourceCommonEnv.sh
+
+# Override SLACK_CONFIG for rebalancer to avoid noise on Slack due to the frequent restarts (every 6 secs)
+echo "SLACK_CONFIG=" >> ${env_file}
+
+# Set the bot identifier
+echo "BOT_IDENTIFIER=LISK_ACROSS_REBALANCER"  >> ${env_file}
+
+# Relaying OFF rebalancing ON
+echo "SEND_RELAYS=false" >> ${env_file}
+echo "SEND_REBALANCES=true"  >> ${env_file}
+
+# Looping mode OFF
+echo "POLLING_DELAY=0" >> ${env_file}
+
+echo "All env vars are set."
+
+# Restart rebalancer every 5 mins
+sleep 300
```

Comment:
- For my understanding, where is this reflected that rebalancer is restarted every 5 minutes?
We just sleep here for 5 minutes.

- `Override SLACK_CONFIG for rebalancer to avoid noise on Slack due to the frequent restarts (every 6 secs)` Why is here mentioned 6 seconds if rebalancer is restarted every 5 minutes?

Reply:
Clarified offline!

**GIST**:

- The rebalancer currently runs in the no loop mode (due to `POLL_DELAY=0`). This means that the rebalancer runs and exits immediately after ~6-10secs
- The docker-compose is configured to `restart:always` for rebalancer as soon as the application exits
- So, to prevent unnecessary noise from these frequent restarts, the SLACK_CONFIG is overridden to `null` for the rebalancer
- And, the Across team advised to run the rebalancer every 5 mins; hence, the sleep within the rebalancer start scripts combined with the `restart:always` config for rebalancer within the docker-compose.yml help us achieve this effect

Reply:
Thanks @sameersubudhi!
It's all clear now!